### DISCUSSION TITLE
Option to download Public Talk Song

### DISCUSSION THREAD

**Discussion by Mercu7io** [sircharlo/meeting-media-manager#94] [Ideas] [answered]
Hello,
First of all, thank you for this really useful tool. I like it a lot!!!!
Maybe I didn't find how to do it, but if I want to download the PT song, is it possible to add a box  (in main page) to choose the song number and it will be downloaded with all media for the week?
Thank you

**Comment by sircharlo** [comment, marked as answer]
Hi there! I'm glad you like it.
Did you check the [documentation](https://github.com/sircharlo/jw-meeting-media-fetcher#configuration)? The option you're looking for is under Configuration > **Offer to import additional media**.
Let me know if you have any more questions!
COS

**Comment by rickinhissmart** [comment]


**Reply by sircharlo** [reply]
As opposed to the current mechanism of selecting the song to download (along with any other additional media) after clicking **“Get media!”**? I’m not sure I can justify the change, if the end result is essentially the same. Perhaps there’s something I’m not seeing?

**Reply by Mercu7io** [reply]
I just wanted all the media in one page. since the PT song have always to be downloaded, I thought it was a nice idea.
But I understand that small changes may not be in the same spirit as the dev and it might be be a lot off work for a simple thing.
Thank you

**Reply by sircharlo** [reply]
Yeah, I understand what you mean @Mercu7io and @rickinhissmart. A field to choose the public talk's song number on the main page, similar to SoundBox's main page. I might go this direction if and when we start holding meetings in person rather than over Zoom, as by then the needs for this app will have changed somewhat. I think for now though, I'd rather keep the layout the way it is. My thinking is, media files are separated into two "categories": media automatically extracted from the meeting's source material, and additional media added by user input.
Does the current layout cause any difficulties for either of you, or is there perhaps a step missing in my logic?

**Reply by sircharlo** [reply]
The "easy song auto-fetch" feature you're describing is already implemented. I think you might have a previous version of the app @rickinhissmart , one from before when that feature was fully implemented. The drag/drop method you're describing is now obsolete for songs, as well as JWPUB files. 

See what I mean: 

![74iIGvYSDG](https://user-images.githubusercontent.com/1275666/132408450-2324fdd2-d574-41d2-a762-f429864c5699.gif)

1. Make sure you are on version 0.9.0 (latest)
2. Make sure option "Offer to import additional media" is enabled
2. Click on blue "Get media" button
2. When presented with "additional media" screen, click on target meeting
3. Click on song and choose the needed song
4. JWMMF downloads and saves the song for you
